lld/COFF/Symbols.h

```diff
@@ -219,7 +219,9 @@ class DefinedRegular : public DefinedCOFF {
     return s->kind() == DefinedRegularKind;
   }
 
-  uint64_t getRVA() const { return (*data)->getRVA() + sym->Value; }
+  uint64_t getRVA() const {
+    return (*data)->getRVA() + (sym->Value != 0xC0000040 ? sym->Value : 0);
```

Comment:
I wonder if we could avoid special-casing that value. From what I can see with binutils, value is simply ignored for such symbols. Maybe we could just always create a symbol with 0 value when handling `isEmptySectionDeclaration` in lld-link.

Reply:
Yes, we most probably should avoid hardcoding it, especially like this; this was my mildly provocative PoC to show where/how we may need to work around it :-)

Indeed, ignoring the offset when handling `isEmptySectionDeclaration` probably is good, but that's apparently not entirely enough either. Note this section symbol:
```
  - Name:            '.idata$2'
    Value:           3221225536
    SectionNumber:   1
    SimpleType:      IMAGE_SYM_TYPE_NULL
    ComplexType:     IMAGE_SYM_DTYPE_NULL
    StorageClass:    IMAGE_SYM_CLASS_SECTION
```

So apparently we should ignore the `Value` entirely, for any section type symbol it seems? I haven't quite yet had time to work out the logic around this, how binutils and link.exe operate around this.